HW#13/Deck of cards.py

```diff
@@ -0,0 +1,39 @@
+import random
+
+
+class Card:
+    number_list = ["Two", "Three", "Four", "Five", "Six", "Seven", "Eight",
+                   "Nine", "Ten", "Jack", "Queen", "King", "Ace"]
+    mast_list = ["Spades", "Clubs", "Diamonds", "Hearts"]
+
+    def __init__(self, number_list, mast_list):
+        self.number_list = number_list
+        self.mast_list = mast_list
+
+
+class CardsDeck:
+
+    def __init__(self):
+        self.deck = []
+        self.number_list = Card.number_list
+        self.mast_list = Card.mast_list
+        for i in self.number_list:
+            for j in self.mast_list:
+                self.deck.append(f'{i} of {j}')
```

Comment:
Better to store a card as an object, not as a string, so you will have a list with objects
```self.cards = [Card(number, mast) for mast in Card.mast_list for number in Card.number_list]```